tests/validators.py

```diff
@@ -69,7 +69,9 @@ def validate_storage():
     """
     Validate storage by creating a PVC.
     """
-    wait_for_pod_state("", "kube-system", "running", label="k8s-app=hostpath-provisioner")
+    wait_for_pod_state(
+        "", "kube-system", "running", label="app.kubernetes.io/name=hostpath-provisioner"
```

Comment:
This is causing the test-upgrade to fail. The test upgrade does the following:

  1. installs the old snap
  2. enables some addons from the old snap including the storage
  3. runs the validations for the enabled addons
  4. refreshes to the new snap
  5. runs the validations again to make sure nothing broke

In the old storage addon that is enabled during the test-upgrade the label is `label="k8s-app=hostpath-provisioner"` so the new validator will have to check the old label as well as the new one.